.github/autolabeler.yml

```diff
@@ -0,0 +1,31 @@
+'accessibility': []
+'addon: a11y': ["addons/a11y/**"]
+'addon: actions': ["addons/actions/**"]
+'addon: backgrounds': ["addons/backgrounds/**"]
+'addon: centered': ["addons/centered/**"]
+'addon: events ': ["addons/events/**"]
```

Comment:
idk if it's critical, but there are few entries with the extra space at the end

Reply:
I wonder if there is some kind of yamllint =D